components/gitpod-db/src/typeorm/migration/1612448906950-ExampleRepositories.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class ExampleRepositories1612448906950 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        const oldEntries = [
+            { url: 'https://github.com/gitpod-io/django-locallibrary-tutorial', description: '**Python** - Tutorial "Local Library" website written in Django', priority: 10 },
+            { url: 'https://github.com/gitpod-io/gs-spring-boot.git', description: '**Java** - Building an Application with Spring Boot', priority: 9 },
+            { url: 'https://github.com/gitpod-io/NextSimpleStarter.git', description: '**JavaScript** - Simple PWA boilerplate with Next.js and Redux', priority: 8 },
+            { url: 'https://github.com/gitpod-io/go-gin-app.git', description: '**Go** - A simple web app implemented in Go and Gin', priority: 7 },
+            { url: 'https://github.com/gitpod-io/rails_sample_app', description: '**Ruby on Rails** - Tutorial sample application', priority: 6 },
+            { url: 'https://github.com/gitpod-io/symfony-demo.git', description: '**PHP** - Symfony Demo Application', priority: 5 },
+            { url: 'https://github.com/theia-ide/theia.git', description: '**Typescript** - Deep dive into Gitpod\\\'s open-source IDE, Theia.', priority: 4 }
+        ];
+        const newEntries = [
+            { url: 'https://github.com/gitpod-io/sveltejs-template', description: '**JavaScript** – A project template for Svelte applications', priority: 90 },
+            { url: 'https://github.com/eclipse-theia/theia', description: '**TypeScript** – A cloud & desktop IDE framework', priority: 80 },
```

Comment:
**issue**: I still find _Theia_ not a good starter example project for users to try on their first time using Gitpod but let's leave it here as a good TypeScript example until we have a better one.

Reply:
Many thanks for your feedback. I'd love to discuss your thoughts on Theia-as-an-example a bit more.

Additional data point: Theia seems to be most frequent "first-time repository" for Gitpod users.

<img width="350" alt="Screenshot 2021-02-09 at 10 59 20" src="https://user-images.githubusercontent.com/599268/107347450-1f87e280-6ac6-11eb-9e88-4ee44ce8f4e3.png">

I'm not sure whether that's more because of the Gitpod button on https://www.theia-ide.org or Theia's presence in Gitpod's examples, or whether that means it's a good starter example for users or not, but I thought this data was relevantHERE.

(FYI, number 2 is the https://gitlab.com/crio-bytes/me_http workshop and number 3 is the Gitpod integration in https://www.circuito.io/ )

Reply:
Thanks for sharing these data points here @jankeromnes! 

**thought**: I could be wrong but I think it seems unlikely that first time users open the dashboard and scroll down to the bottom of the example projects list which spans below the fold (outside the viewport) to start a dev environment for Theia. 💭